app/assets/javascripts/analytics/_events.js

```diff
@@ -118,10 +136,10 @@
       });
     },
     'init': function () {
-      // TODO: this should be within a specific class and only initialised on the supplier list download page
+      // TODO: this should be within a specific class and only initialised on the supplier list download page (for supplierListDownload)
+      // and the opportunities list page (for opportunityDataDownload)
       // Currently this is running on click of any <a> link!
-      $('body').on('click', 'a', this.supplierListDownload);
-      $('body').on('click', '[data-analytics=trackEvent]', this.trackEvent);
+      $('body').on('click', 'a', this.supplierListDownload).on('click', 'a', this.opportunityDataDownload).on('click', '[data-analytics=trackEvent]', this.trackEvent);
```

Comment:
Might look better like this for readability:
```
$('body')
    .on('click', 'a', this.supplierListDownload)
    .on('click', 'a', this.opportunityDataDownload)
    .on('click', '[data-analytics=trackEvent]', this.trackEvent);
```

Reply:
Cool, will move this like you suggested, thanks! :)